addons/im_livechat/report/im_livechat_report_channel.py

```diff
@@ -169,79 +85,130 @@ def _select(self) -> SQL:
                 to_char(date_trunc('hour', C.create_date), 'YYYY-MM-DD HH24:MI:SS') as start_date_hour,
                 to_char(date_trunc('hour', C.create_date), 'HH24') as start_hour,
                 EXTRACT(dow from C.create_date)::text AS day_number,
-                EXTRACT('epoch' FROM MAX(message_vals.last_message_dt) - c.create_date)/60 AS duration,
+                EXTRACT('epoch' FROM message_vals.last_message_dt - c.create_date)/60 AS duration,
```

Comment:
we should probably improve this condition at some point with a real field because posting a message on channel after the end of the session will mess up the stats

Reply:
Agreed, in the meantime it's not that big of a deal as it's not possible from the UI neither for the visitor or the operator.